initfiles/bin/init_roxie

```diff
@@ -40,9 +40,6 @@ fi
 export LIBC_FATAL_STDERR_=1
 export restarts=0
 
-ulimit -c unlimited
-ulimit -n $NUM_ROXIE_HANDLES
```

Comment:
Problem is.. we used to have a per roxie way of handling this, now we have a universal default.
But no way to configure it (I think).

The defaults (now all in hpcc_common.in), would be better in something like environment.conf..

@richardkchapman - what do you think?

I suspect can wait for another PR.